app/helpers/waste_exemptions_engine/plurals_helper.rb

```diff
@@ -5,7 +5,7 @@ module PluralsHelper
     def confirmation_comms(form)
       emails = [form.applicant_email, form.contact_email]
       emails.uniq!
-      emails.delete(WasteExemptionsEngine.configuration.assisted_digital_email)
+      emails.delete(nil)
```

Comment:
Does this also need to handle blank emails, i.e. empty strins values?

Reply:
I don't think so, because you can't enter a blank string or it will give a validation error when your entering an email. So with that do I still need to include it as a possibility?

Reply:
`contact_email` is being set to an empty string in WCR using similar code. However the WCR code handles both blank and nil email addresses, so it's more something to tidy up rather than a defect - Tim has created https://eaflood.atlassian.net/browse/RUBY-2028 to fix this later. I don't think we can rule out blank emails here, so best allow for the possibility in the code which refers to the email address attributes.